src/agent.rs

```diff
@@ -82,7 +76,27 @@ impl Agent {
         }
 
         // Add specific args.
-        for arg in &args {
+        // Modify cipher arguments to match the format required by the different shims. 
+        let mut cipher_arg=false;
+        for a in &args {
```

Comment:
This arg parsing needs rework... But not in this PR.

Reply:
I will work on this in the future and opened an issue for it. https://github.com/jallmann/tls-interop/issues/11